code/game/objects/items/weapons/storage/book.dm

```diff
@@ -79,22 +80,13 @@
 			ticker.mode.remove_cultist(M.mind)*/
 		if ((istype(M, /mob/living/carbon/human) && prob(60)))
 			bless(M)
-			if(ishuman(M))
-				var/mob/living/carbon/human/H = M
-				var/message_halt = 0
-				for(var/obj/item/organ/limb/affecting in H.organs)
-					if(affecting.status == ORGAN_ORGANIC)
-						if(message_halt == 0)
-							M.visible_message("<span class='notice'>[user] heals [M] with the power of [src.deity_name]!</span>")
-							M << "<span class='boldnotice'>May the power of [src.deity_name] compel you to be healed!</span>"
-							playsound(src.loc, "punch", 25, 1, -1)
-							message_halt = 1
-					else
-						user << "<span class='warning'>[src.deity_name] refuses to heal this metallic taint!</span>"
-						return
-
-
-
+			if(bless(M))
+				M.visible_message("<span class='notice'>[user] heals [M] with the power of [src.deity_name]!</span>")
+				M << "<span class='boldnotice'>May the power of [src.deity_name] compel you to be healed!</span>"
+				playsound(src.loc, "punch", 25, 1, -1)
+			else
+				user << "<span class='warning'>[src.deity_name] refuses to heal this metallic taint!</span>"
```

Comment:
This isn't quite right. If you try to heal someone at max health (bless() returns false), it gives the metallic message.